### PR TITLE
use generate_cases

### PR DESCRIPTION
**Pull request by czue**
@benrudolph cc @esoergel

### CODE REVIEW

File: corehq/apps/userreports/tests/test_transforms.py [hunk 1]
```diff
@@ -35,20 +36,20 @@ def test_int(self):
...
 
 
+@generate_cases((
+    ('11', '11'),
+    ('11.23', '11'),
+    ('notanumber', 'notanumber'),
+    ('', ''),
+), NumberFormatTransformTest)
+def test_number_format_transform_strings(self, input, expected_result):
+    self.assertEqual(expected_result, self.transform(input))
```
Comment (esoergel) [inline corehq/apps/userreports/tests/test_transforms.py]: Do all our various test runners pick this up now?
- [ ] Travis
- [ ] Nose
- [ ] `./manage.py test`

Comment (czue) [inline corehq/apps/userreports/tests/test_transforms.py]: I didn't try nose, but the others definitely do. If you declare it like this it just adds each case as a test to the `NumberFormatTransformTest`.

Comment (benrudolph) [inline corehq/apps/userreports/tests/test_transforms.py]: @millerdev wrote it so i'd think it'd run on nose as well

Comment (esoergel) [inline corehq/apps/userreports/tests/test_transforms.py]: Oh, I see, I didn't realize it modifies the test class.  That makes sense.

**Pull request merged**